README.md

```diff
@@ -51,13 +51,13 @@ yarn run dev
 </details>
 <p></p>
 
-Your application is running at http://localhost:3000/.
+Your application will be running at http://localhost:3000/.
 
 ### Creating a new app with a code example
 
-You can create a new application with an example code using `create-resolve-app` with the `-e` flag followed by an example name. Use the `create-resolve-app -h` command to list the available [examples](./examples/).
+You can create a new application with a code example using `create-resolve-app` with the `-e` flag followed by the example's name. Use the `create-resolve-app -h` command to list the available [examples](./examples/).
```

Comment:
There are examples - serveral of them, in the folder "examples". You can include the code of the example into your project. So I would use "example code" or even "code of the specific example".

Reply:
The phrase "example code" looks unfamiliar to me, so I'm not sure if it can be used like this (a native speaker would know better). How about this then?

> There are several [code examples](./examples/) provided. You can add code from these examples into your projects. To list the available examples use the `create-resolve-app -h` command. 
> 
> To create a new application with the code of the specific example use the `create-resolve-app` command with the `-e` flag followed by the example's name.